packages/lesswrong/components/sunshineDashboard/ModeratorUserInfo/UserReviewStatus.tsx

```diff
@@ -6,7 +6,9 @@ const styles = (theme: ThemeType): JssStyles => ({
   root: {
     marginTop: 16,
     fontStyle: "italic"
-  }
+  },
+  accountFlag: {
+  },
```

Comment:
Unused?

Reply:
I'm not sure what our convention should be in cases like this, where a classname is applied, and seems like someone might want to apply styles, but there aren't any actual styles yet. Seems like it could go either way.